README.md

```diff
@@ -119,9 +119,7 @@ func main() {
 	}
```

Comment:
Not your fault, but we also need to change the README example code PrimaryPurpose definition to be a slice or this example won't build.

Suggestion:
```
PrimaryPurpose:   []sbom.Purpose{sbom.Purpose_APPLICATION},
```

Reply:
Sorry for not testing the example before submitting it. It as really a last minute improvement I did to the PR. At least it helped us catch a bug :). It runs now:

```bash
go run test.go
```

```json
{
    "spdxVersion": "SPDX-2.3",
    "dataLicense": "CC0-1.0",
    "SPDXID": "SPDXRef-DOCUMENT",
    "name": "",
    "documentNamespace": "https://spdx.org/spdxdocs/",
    "creationInfo": {
        "licenseListVersion": "3.20",
        "creators": [
            "Tool: protobom-devel",
            "Tool: ACME SBOM Tool-1.0"
        ],
        "created": "2023-10-21T00:16:25Z"
    },
    "packages": [
        {
            "name": "My Software Name",
            "SPDXID": "SPDXRef-pkg:generic/my-software@v1.0.0",
            "versionInfo": "v1.0.0",
            "downloadLocation": "NOASSERTION",
            "filesAnalyzed": false,
            "licenseConcluded": "Apache-2.0",
            "licenseComments": "Apache License",
            "primaryPackagePurpose": "APPLICATION"
        }

```